tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -131,6 +131,104 @@ var _ = Describe("odo devfile push command tests", func() {
 			Expect(statErr).ToNot(HaveOccurred())
 		})
 
+		It("checks that odo push works with a devfile with composite commands", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, cmpName)
+
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfileCompositeCommands.yaml"), filepath.Join(context, "devfile.yaml"))
+
+			output := helper.CmdShouldPass("odo", "push", "--context", context)
+			Expect(output).To(ContainSubstring("Executing mkdir command"))
+
+			// Verify the command executed successfully
+			var statErr error
+			podName := cliRunner.GetRunningPodNameByComponent(cmpName, namespace)
+			cliRunner.CheckCmdOpInRemoteDevfilePod(
+				podName,
+				"runtime",
+				namespace,
+				[]string{"stat", "/projects/testfolder"},
```

Comment:
This will ends up with error on windows. Please use something like https://github.com/openshift/odo/blob/master/tests/integration/cmd_push_test.go#L289

Reply:
Can you please also run your tests on windows once

Reply:
@prietyc123 why would that fail on Windows? That’s verifying the path of a folder in a remote Linux container, so Unix paths need to be used here.

Reply:
@prietyc123 The tests I added run fine on Windows without modifciation